docker-compose.yml

```
@@ -37,6 +37,7 @@ services:
   dev-ticker:
     image: "rsksmart/rollup-dev-ticker:1.0.0-beta"
```

```suggestion
    image: "rsksmart/rollup-dev-ticker:1.1.0-beta"
```

@antomor we may include this change in this commit and then create the image in main, or just approve it and merge it, or create a new PR, so we can run the tests with github action.

I already ran the tests and everything is working as expected. :)

---

New version published 👍🏼